src/index.html

```diff
@@ -8,7 +8,43 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images/logo.png"
+        alt="MOYO logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul  class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Laptops&Computers" class="nav__link data-qa">Laptops & Computers</a>
```

Comment:
```suggestion
            <a href="#Laptops&Computers" class="nav__link" data-qa="hover">Laptops & Computers</a>
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96265844/b660e24a-daa9-409c-baef-c0f0fb0948b3)